### ISSUE TITLE
Fails to activate

### ISSUE THREAD

**Issue by simenkva** [hesstobi/atom-autocomplete-latex-cite#18]
The package fails to activate upon installation. 

**Atom**: 1.37.0 x64
**Electron**: 2.0.18
**OS**: Mac OS X 10.14.5
**Thrown From**: [autocomplete-latex-cite](https://github.com/hesstobi/atom-autocomplete-latex-cite) package 0.3.2

### Stack Trace

Failed to activate the autocomplete-latex-cite package

```
At Cannot find module './ts-compiled/command-aliases'

Error: Cannot find module './ts-compiled/command-aliases'
    at Module._resolveFilename (module.js:543:15)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:154765)
    at Module.require (/app.asar/static/index.js:40:43)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146072)
    at /packages/autocomplete-latex-cite/node_modules/latex-to-unicode-converter/index.js:6:10)
    at /packages/autocomplete-latex-cite/node_modules/latex-to-unicode-converter/index.js:17:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146782)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:150329)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146072)
    at /packages/autocomplete-latex-cite/lib/lite-bibtex-parse.js:32:31)
    at /packages/autocomplete-latex-cite/lib/lite-bibtex-parse.js:373:40)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146782)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:150329)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146072)
    at /packages/autocomplete-latex-cite/lib/cite-manager.coffee:7:15)
    at /packages/autocomplete-latex-cite/lib/cite-manager.coffee:1:1)
    at /packages/autocomplete-latex-cite/lib/cite-manager.coffee:1:1)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146782)
    at Object.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:150329)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146072)
    at /packages/autocomplete-latex-cite/lib/cite-provider.coffee:1:15)
    at /packages/autocomplete-latex-cite/lib/cite-provider.coffee:1:1)
    at /packages/autocomplete-latex-cite/lib/cite-provider.coffee:1:1)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146782)
    at Object.value [as .coffee] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:150329)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:504:12)
    at Function.Module._load (module.js:496:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146072)
    at Object.provide (/packages/autocomplete-latex-cite/lib/main.coffee:20:22)
    at Package.activateServices (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3424358)
    at Package.activateNow (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3421479)
    at string.e.trim.length.activationHookSubscriptions.add.packageManager.onDidTriggerActivationHook (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:2445)
    at Function.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1174290)
    at Emitter.emit (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1175731)
    at deferredActivationHooks.triggeredActivationHooks.forEach.e (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:382852)
    at Set.forEach (<anonymous>)
    at PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:382814)
    at /Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:901281
    at r (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:895887)
    at n (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:1043884)
    at ChildProcess.o.process.on.e (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:1044090)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
```

### Commands

```
     -2:35.9.0 core:cancel (input.hidden-input)
  2x -2:35.4.0 core:move-down (input.hidden-input)
  2x -2:34.3.0 core:move-up (input.hidden-input)
     -2:33.6.0 core:move-down (input.hidden-input)
     -2:31.3.0 core:save (input.hidden-input)
     -2:18.6.0 intentions:show (input.hidden-input)
  2x -2:17.6.0 core:move-down (input.hidden-input)
     -2:14.6.0 core:move-up (input.hidden-input)
     -2:14.2.0 intentions:confirm (input.hidden-input)
  3x -2:13.2.0 core:move-down (input.hidden-input)
     -2:11 core:save (input.hidden-input)
 42x -2:07.2.0 core:move-up (input.hidden-input)
 58x -2:05.1.0 core:move-down (input.hidden-input)
 14x -2:02.3.0 core:move-up (input.hidden-input)
  2x -1:47.3.0 linter-ui-default:toggle-panel (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-atom-dark-syntax.theme-atom-dark-ui)
     -0:53 core:backspace (input.hidden-input)
```

### Non-Core Packages

```
atom-latex 0.8.5 
atom-terminal-panel 4.4.4 
autocomplete-latex-cite 0.3.2 
busy-signal 2.0.1 
intentions 1.1.5 
language-latex 1.2.0 
latex 0.50.2 
linter 2.3.0 
linter-spell 0.15.0 
linter-spell-latex 0.11.0 
linter-ui-default 1.7.1 
multi-wrap-guide 1.1.6 
script 3.18.1 
```

**Comment by hesstobi** [comment]
The issue here is that the https://www.npmjs.com/package/latex-to-unicode-converter is gone from npm. I need to find a alternative. I may look on weekend on this.

**Comment by nateraluis** [comment]
Any update on this? I'm also having the same issue.

**Comment by mayhewsw** [comment]
Pull request #20 solves this issue.

**Comment by EnochMakenp** [comment]
I manually applied the fix for #8 from @mayhewsw and installed the new latex-to-unicode manually, and the autocomplete-latex-cite works now :-)
I've installed Atom 1.39.1 just a few days ago, and I'm on Windows 10.
Thanks !

**Comment by hesstobi** [comment]
Fixed in version v0.3.4
